server/middleware/validateInfo.js

```diff
@@ -8,16 +8,25 @@ const validateInfo = (req,res,next)=>{
     if(req.path === "/register"){
         
         if(![email,name,password].every(Boolean)){
+
```

Comment:
This is just a recommendation! When you have these if else statements that are one-liners, you should format them like this:
`if(!this.cryptoToEur.contains(transaction.getCurrency())) {market = this.end_fiat_currency;}`
Don't look at the code its from my project xD, but you get the point